docs/Linuxbrew.md

```diff
@@ -0,0 +1,99 @@
+![Linuxbrew logo](https://linuxbrew.sh/images/linuxbrew-256x256.png)
+
+# Linuxbrew
+
+The Homebrew package manager may be used on Linux and Windows 10+, using [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/about), as well as macOS. Homebrew is referred to as Linuxbrew when running on Linux or Windows. It can be installed in your home directory and does not require root access. Recent versions of *glibc* and *gcc* may be installed in your home directory on an older distribution of Linux. A Linuxbrew installation does not use any libraries provided by your host system, except *glibc* and *gcc* if they are sufficiently current.
+
+[Features](#features), [dependencies](#dependencies) and [installation instructions](#install) are described below. Terminology (e.g. the difference between a Cellar, Tap, Cask and so forth) is [explained in the documentation](https://docs.brew.sh/Formula-Cookbook#homebrew-terminology).
+
+# Install
+
+The installation script installs Linuxbrew to `/home/linuxbrew/.linuxbrew` if possible and in your home directory at `~/.linuxbrew` otherwise. Using `/home/linuxbrew/.linuxbrew` allows the use of more binary packages, called bottles, than installing in your personal home directory.
+
+Paste at a Terminal prompt:
+
+```sh
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"
+```
+
+Follow the *Next steps* instructions to add Linuxbrew to your `PATH` and to your bash shell profile script, either `~/.profile` on Debian/Ubuntu or `~/.bash_profile` on CentOS/Fedora/RedHat.
+
+```sh
+test -d ~/.linuxbrew && eval $(~/.linuxbrew/bin/brew shellenv)
+test -d /home/linuxbrew/.linuxbrew && eval $(/home/linuxbrew/.linuxbrew/bin/brew shellenv)
```

Comment:
it just occurred to me... `-d` tests if directory exists. We need to test `-d`, `-w`, and `-x`

Reply:
If the directory exists, I prefer the user get an error message if it's not also `-w` or `-x`.

Reply:
What if neither exist?

Reply:
These commands are meant to be run after a successful run of `sh -c "$(curl -fsSL https://raw.githubusercontent.com/Linuxbrew/install/master/install.sh)"`, after which one of these two directories will exist.